brpscripts/ALT/brp-adjust_libraries.in

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -e
+#
+# brp-adjust_libraries - update symbolic links to shared libraries.
+#
+# Copyright (C) 2000,2003,2005  Dmitry V. Levin <ldv@altlinux.org>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+#
+
+. @RPMCONFIGDIR@/functions
+ValidateBuildRoot
+
+cd "$RPM_BUILD_ROOT"
+
+libs=`find -type d -name lib -o -name lib64`
+
+if [ -n "$libs" ]; then
+	echo "Adjusting library links in $RPM_BUILD_ROOT"
+	/sbin/ldconfig -nv $libs
```

Comment:
this one is very similar to https://github.com/rpm-software-management/rpm-extras/blob/master/brpscripts/Fedora/brp-ldconfig ;)

Reply:
Yes, ALT had this years before Fedora.